docs/static/running-logstash-command-line.asciidoc

```diff
@@ -223,5 +223,17 @@ With this command, Logstash concatenates three config files, `/tmp/one`, `/tmp/t
   as the log4j logging configuration. This can also be set through the LS_SETTINGS_DIR environment variable.
   The default is the `config` directory under Logstash home.
 
+*`--enable-local-plugin-development`*::
+[NOTE]
+====
+This flag is for Logstash developers only, and should not need to be set by end users.
+====
+This flag enables a developer to update their local Gemfile, without running into issues caused by a
+frozen lockfile. This flag would typically be used when developing/testing plugins locally
+
+
+
+
```

Comment:
Suggestion for your consideration. It looks/reads better when the description comes before the note.  We can use the basic `Note` formatting here because it's a single line.

Reply:
I tried to remove the extra lines in the suggestion, but it doesn't seem to be reflecting that.  Regardless, no need for extra lines. :-)